src/commands/config/dump.js

```diff
@@ -0,0 +1,36 @@
+import inquirer from 'inquirer'
+import configDump from '../../config-dump'
+import * as load from '../../util/load'
+import merge from 'lodash.merge'
+
+export const command = 'dump [env]'
+export const desc = 'Prints all common environmental variables and differences'
+export function builder (yargs) {
+  return yargs
+  .pkgConf('shep', process.cwd())
+  .boolean('json')
+  .describe('json', 'Formats output as JSON')
+  .example('shep config dump beta', 'Print to console all environment variables of environment `beta` in JSON format')
+}
+
+export async function handler (opts) {
+  const envs = await load.envs()
```

Comment:
Is this still loading from API gateway? I think now it needs to be looking at the func aliases and deduping